diff -burNp remind-03.01.05/src/globals.h remind-03.01.05_new/src/globals.h
--- remind-03.01.05/src/globals.h	2008-04-15 03:11:07.000000000 +0200
+++ remind-03.01.05_new/src/globals.h	2008-09-24 10:00:04.951585836 +0200
@@ -12,6 +12,8 @@
 /*                                                             */
 /***************************************************************/
 
+#include "custom.h"
+
 #ifdef MK_GLOBALS
 #undef EXTERN
 #define EXTERN
diff -burNp remind-03.01.05/src/queue.c remind-03.01.05_new/src/queue.c
--- remind-03.01.05/src/queue.c	2008-01-31 03:37:40.000000000 +0100
+++ remind-03.01.05_new/src/queue.c	2008-09-24 10:00:40.504779330 +0200
@@ -10,6 +10,7 @@
 /*                                                             */
 /***************************************************************/
 
+#include "custom.h"
 #include "config.h"
 
 /* Solaris needs this to get select() prototype */
diff -burNp remind-03.01.05/src/rem2ps.h remind-03.01.05_new/src/rem2ps.h
--- remind-03.01.05/src/rem2ps.h	2007-08-08 18:01:20.000000000 +0200
+++ remind-03.01.05_new/src/rem2ps.h	2008-09-24 10:01:17.422479679 +0200
@@ -10,6 +10,8 @@
 /*                                                             */
 /***************************************************************/
 
+#include "custom.h"
+
 char *PSProlog1[] =
 {
     "% This file was produced by Remind and Rem2PS, written by",
diff -burNp remind-03.01.05/src/types.h remind-03.01.05_new/src/types.h
--- remind-03.01.05/src/types.h	2008-03-25 01:22:22.000000000 +0100
+++ remind-03.01.05_new/src/types.h	2008-09-24 09:59:17.236995290 +0200
@@ -11,6 +11,7 @@
 /***************************************************************/
 
 #include <limits.h>
+#include "custom.h"
 #include "dynbuf.h"
 
 /* Values */
